sodetlib/det_config.py

```diff
@@ -83,15 +88,18 @@ def odict_rep(dumper, data):
     "lms_gain": 0, "feedback_gain": 2048, "frac_pp": None, "lms_freq_hz": None,
 
     # Gradient Descent Parameters
-    "gradientDescentMaxIters": 150,
+    "gradientDescentMaxIters": 100,
     "gradientDescentAverages": 2, 
     "gradientDescentGain": 0.001,
     "gradientDescentConvergeHz": 500,
     "gradientDescentStepHz": 5000,
     "gradientDescentMomentum": 1,
     "gradientDescentBeta": 0.1
 }
-bg_defaults = {}
+bg_defaults = {
+    'overbias_voltage': 19.9,
+    'cool_voltage': 10.0
```

Comment:
Currently this is used in the `overbias_dets` function added to sodetlib.util to be the voltage that dets should be set to after overbiasing, if you just want detectors to be normal. I think that value should be saved in the dev cfg, though maybe cool_voltage is not the right name for it.

Reply:
Okay I think that makes sense.
I'm only used to `cool_voltage` being a kwarg in taking IVs, where `cool_voltage` is optionally applied during the `cool_wait` period after overbiasing, otherwise the `bias_high` value is applied during that time and then the IVs start from there.
For the 30 or so seconds of `cool_wait` we didn't see a big difference in going from 18 V down to 12 V or so, but maybe you will if just hanging out normal for long periods of time.

Reply:
Ok I'm going to keep cool_voltage at 10 V but will keep my eye on it. If we think it should be changed I'm happy to update the value or implementation